.travis.yml

```diff
@@ -0,0 +1,95 @@
+    - mysql-client-5.6
+
+cache:
+  directories:
+    - $HOME/.composer/cache/files
+
+env:
+  global:
+    # For functional and acceptance tests
+    - COMPOSE_FILE="doc/docker/base-dev.yml:doc/docker/selenium.yml"
+    # In order to specify extra flags like ignoring platform reqs, using only stable packages and so on.
+    - COMPOSER_FLAGS=""
+    - EZPLATFORM_REPO="https://github.com/ezsystems/ezplatform.git"
+
+matrix:
+  fast_finish: true
+  include:
+# 7.1
+    - name: "PHP 7.1 Unit tests"
+      php: 7.1
+      env: TEST_CONFIG="phpunit.xml"
+    - name: "PHP 7.1 REST Integration Tests"
+      php: 7.1
+      env: REST_TEST_CONFIG="phpunit-integration-rest.xml" SYMFONY_ENV=behat PHP_IMAGE=ezsystems/php:7.1-v1 SYMFONY_DEBUG=1 SF_CMD="ez:behat:create-language 'pol-PL' 'Polish (polski)'"
+# 7.2
+    - name: "PHP 7.2 Unit tests"
+      php: 7.2
+      env: TEST_CONFIG="phpunit.xml"
+    - name: "PHP 7.2 REST Integration Tests"
+      php: 7.2
+      env: REST_TEST_CONFIG="phpunit-integration-rest.xml" SYMFONY_ENV=behat PHP_IMAGE=ezsystems/php:7.2-v1 SYMFONY_DEBUG=1 SF_CMD="ez:behat:create-language 'pol-PL' 'Polish (polski)'"
+# 7.3
+    - name: "PHP 7.3 Unit tests"
+      php: 7.3
+      env: TEST_CONFIG="phpunit.xml"
+    - name: "PHP 7.3 REST Integration Tests"
+      php: 7.3
+      env: REST_TEST_CONFIG="phpunit-integration-rest.xml" SYMFONY_ENV=behat PHP_IMAGE=ezsystems/php:7.3-v1 SYMFONY_DEBUG=1 SF_CMD="ez:behat:create-language 'pol-PL' 'Polish (polski)'"
+    - name: "Code Style Check"
+      php: 7.3
+      env: CHECK_CS=1
+
+# test only master, stable branches and pull requests
+branches:
+  only:
+    - master
+    - /^\d.\d+$/
+
+# setup requirements for running unit/integration/behat tests
+before_install:
+  # Disable memory_limit for composer
+  - echo "memory_limit=-1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
+  # Prepare system
+  - if [ "$REST_TEST_CONFIG" != "" ] ; then ./.travis/prepare_ezplatform.sh ; fi
+  # Execute Symfony command if specified
+  - if [ "$SF_CMD" != "" ] ; then cd "$HOME/build/ezplatform"; docker-compose exec --user www-data app sh -c "bin/console $SF_CMD" ; fi
+  # Detecting timezone issues by testing on random timezone
+  - TEST_TIMEZONES=("America/New_York" "Asia/Calcutta" "UTC")
+  - TEST_TIMEZONE=${TEST_TIMEZONES["`shuf -i 0-2 -n 1`"]}
+
+install:
+  - if [ "$CHECK_CS" == "1" -o "$TEST_CONFIG" != "" ]; then travis_retry composer install --prefer-dist --no-interaction --no-suggest ; fi
+
+# execute phpunit or behat as the script command
+script:
+  - if [ "$TEST_CONFIG" != "" ] ; then php -d date.timezone=$TEST_TIMEZONE -d memory_limit=-1 vendor/bin/phpunit -c $TEST_CONFIG ; fi
+  - if [ "$REST_TEST_CONFIG" != "" ] ; then cd "$HOME/build/ezplatform"; docker-compose exec --user www-data app sh -c "php -d date.timezone=$TEST_TIMEZONE -d memory_limit=-1 bin/phpunit --bootstrap vendor/autoload.php -c vendor/ezsystems/ezplatform-rest/$REST_TEST_CONFIG"  ; fi
```

Comment:
Follow up for later: install dependencies for ezplatform-rest and run tests from there.